lib/topological_inventory/orchestrator/worker.rb

```diff
@@ -38,8 +38,8 @@ def api_base_url
           require "uri"
 
           URI::HTTP.build(
-            :host => ENV["TOPOLOGICAL_INVENTORY_API_SERVICE_HOST"],
-            :port => ENV["TOPOLOGICAL_INVENTORY_API_SERVICE_PORT"],
+            :host => ENV["SOURCES_API_SERVICE_HOST"],
+            :port => ENV["SOURCES_API_SERVICE_PORT"],
```

Comment:
This will need a config change because the sources service isn't in the same namespace.

Also based on [the existing env vars](https://github.com/RedHatInsights/e2e-deploy/blob/master/env/ci.yml#L6-L15) I think the convention would be `SOURCES_HOST` and `SOURCES_PORT`

Reply:
:+1: done